resources/js/processes/modeler/components/inspector/StageManager.vue

```diff
@@ -109,15 +111,16 @@ const updateStagesForAllFlowConfigs = (stages) => {
 
 const removeStageInAllFlowConfig = (stage) => {
   const links = getModeler().graph.getLinks();
-  for (const link of links) {
+  links.forEach((link) => {
```

Comment:
This change is compatible, but keep in mind that **for** in its native form is significantly faster than **forEach**. With arrays that have many elements, the difference is very noticeable.